src/components/calendar/Calendar.sass

```diff
@@ -0,0 +1,24 @@
+.fc .fc-toolbar-title 
+    font-size: 20px
+    margin: 0
+
+  
+.fc .fc-button-primary
+    border-radius: 7px
+    user-select: none
+    border-color: #0082ca
+    background-color: #0082ca
+    transition: .6s cubic-bezier(.25,.8,.25,1)
+
+.fc .fc-button-primary:hover
+    background-color: #005b8d
+    border-color: #005b8d
+
+.fc .fc-button-primary:not(:disabled).fc-button-active
+    background-color: #004b74
+    border-color: #004b74
+
```

Comment:
3 diffrents background colors/ Please attach screen with your design

Reply:
I suppose? For hover, active button and by default